tutorials/get-started-deploy-kapsule/index.mdx

```diff
@@ -72,7 +72,7 @@ Before starting the practical steps of this tutorial, we review a few key Kubern
 
 * **Managed Kubernetes**: Managed Kubernetes is when third-party providers take over some of the responsibility for setting up and operating Kubernetes. Scaleway has two managed Kubernetes products: [Kubernetes Kapsule](https://www.scaleway.com/en/kubernetes-kapsule/) and [Kubernetes Kosmos](https://www.scaleway.com/en/kubernetes-kosmos/). Kapsule is our classic Kubernetes offer, where all nodes and their pods in the cluster are managed by Scaleway and hosted on Scaleway Instances. Kosmos allows you to have multi-cloud clusters, with pods hosted on nodes from different cloud providers.
 
-In both cases, Scaleway walks you through the setup of your cluster and manages your control plane for free. All the control plane's components, and kubelets on the nodes, are fully managed. There is no need to connect to your nodes directly, as all actions and configurations can be done from the Scaleway console, via the `kubectl` command, the [Scaleway Kapsule API](https://www.scaleway.com/en/developers/api/kubernetes/) or via [Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/k8s_cluster). You can also monitor your cluster from the Kubernetes dashboard web interface.
+In both cases, Scaleway walks you through the setup of your cluster and manages your control plane for free. All the control plane's components, and kubelets on the nodes, are fully managed. There is no need to connect to your nodes directly, as all actions and configurations can be done from the Scaleway console, via the `kubectl` command, the [Scaleway Kapsule API](https://www.scaleway.com/en/developers/api/kubernetes/) or via [Terraform/OpenTofu](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/k8s_cluster). You can also monitor your cluster from the Kubernetes dashboard web interface.
```

Comment:
Have a link for Terraform and the other for OpenTofu

Reply:
```suggestion
In both cases, Scaleway walks you through the setup of your cluster and manages your control plane for free. All the control plane's components, and kubelets on the nodes, are fully managed. There is no need to connect to your nodes directly, as all actions and configurations can be done from the Scaleway console, via the `kubectl` command, the [Scaleway Kapsule API](https://www.scaleway.com/en/developers/api/kubernetes/), [Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/k8s_cluster), or via [OpenTofu](https://search.opentofu.org/provider/opentofu/scaleway/latest). You can also monitor your cluster from the Kubernetes dashboard web interface.
```